Formula/kamel.rb

```diff
@@ -0,0 +1,38 @@
+class Kamel < Formula
+  desc "Apache Camel K CLI"
+  homepage "https://camel.apache.org/"
+
+  url "https://github.com/apache/camel-k.git",
+    :tag      => "1.0.0-RC2",
+    :revision => "24ddce5afb41c70a0bf30e06f74c6f380f5fd851"
+  head "https://github.com/apache/camel-k.git"
+
+  bottle :unneeded
```

Comment:
It compiles so should be bottled

Reply:
removed the _"bottle :unneeded"_ statement allowing test bot for its creation as per the  [docs](https://docs.brew.sh/Bottles#bottle-dsl-domain-specific-language)